pages/Useful Utilities/File-Managers.md

```diff
@@ -0,0 +1,35 @@
+---
+weight: 10
+title: File Managers
+---
+
+## GUI
+
+- [dolphin](https://github.com/KDE/dolphin): File manager by KDE.
+- [nautilus](https://apps.gnome.org/en/Nautilus/): A file browser for Gnome.
```

Comment:
"by KDE" but "for Gnome"

pick one pls

Reply:
They are the official description in their respective project repo. Change them anyway for consistency?

Reply:
yea